tools/lkl/scripts/checkpatch.sh

```diff
@@ -17,8 +17,8 @@ if [ -z "$upstream" ]; then
 fi
 
 if [ -z "$lkl" ]; then
-    echo "can't find lkl remote, quiting"
-    exit 1
+    git remote add lkl-upstream https://github.com/lkl/linux || true
+    lkl=`git remote -v | grep lkl | cut -f1 | head -n1`
```

Comment:
Should we grep with $LKL here to be as specific as possible?

Reply:
fixed this.